common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,158 @@
+The BLHeli firmware and configuration applications were developed to allow the configuration of ESCs and provide additional features. ESCs with this firmware allow configuring timing, motor direction, LEDs, motor drive frequency, etc.  Before attempting to use BLHeli please follow the :ref:`DShot setup instructions <common-dshot-escs>`.
 
-Most BLHeli ESCs can autodetect if PWM or DShot is selected by the motor protocol parameter set by the user. See the configuration section of :ref:`common-brushless-escs` for more information. In addition, some BLHeli32 capable ESCs offer bi-directional capabilities for telemetry reporting over the control link, see the section below.
+This page includes setup instructions for the following features
 
-.. note::
-   ArduPilot firmware supports the pass-through protocol with up-to-date BLHeli_32 firmware and BLHeliSuite32, or BLHeli_S firmware and BLHeliSuite only.
+- Pass-Through support allows the BLHeli application to be used to configure the ESCs while remaining connected to the autopilot
+- :ref:`Reversible DShot <blheli32-reversible-dshot>` (aka 3D mode) allows the motor to be spun in either direction
+- :ref:`Bi-directional Dshot <bidir-dshot>` allows the ESCs to send RPM back to the autopilot without the need for an additional telemetry connection
+- :ref:`ESC Telemetry <blheli32-esc-telemetry>` allowing the ESCs to send RPM, voltage and current information back to the autopilot so that it can be logged, viewed in real-time or even allow the removal of a :ref:`battery monitor <esc-telemetry-based-battery-monitor>`
 
-.. warning:: For pass-through to function, the :ref:`motor protocol <MOT_PWM_TYPE>` (Copter,Rover) or :ref:`Q_M_PWM_TYPE<Q_M_PWM_TYPE>` (QuadPlane) must be set to a digital protocol, ie. one of the DShot protocols. If you wish to use one of the other protocols, just reset the motor protocol after using pass-through to change motor directions or set min/max values. The autopilot must be re-booted after a protocol change. In addition, the safety switch, if used, must be pressed to allow the servo/motor outputs to be active.
+"BLHeli" covers covers multiple (sometimes competing) projects providing ESCs firmware and accompanying configuration software
 
-BLHeli pass-through setup and use:
-----------------------------------
+- BLHeli was the original open source software that is no longer maintained and is not available on modern ESCs
+- `BLHeli32 <https://github.com/bitdump/BLHeli>`__ is closed source and based on 32bit ARM MCUs.  All modern BLHeli ESCs use BLHeli32
```

Comment:
This URL also includes vanilla BLHeli_S (version 16.7 - the last official version)

Reply:
ok, fixed after discussion with @andyp1per